src_new/owt.js

```diff
@@ -57,16 +57,35 @@ window.PWT.sfDisplayPMPCreative = function(theDocument, values, priorityArray){
 };
 
 
-window.OWT = {};
+window.OWT = {
```

Comment:
why we have not used PWT name-space ? 
I had given same review-comment earlier as well.

Reply:
We will also need to ask IAC to make changes when we move OWT to PWT